### PR TITLE
Neo4J-update-t-4.1

### PR DESCRIPTION
**Pull request by MattGalvis**

### CODE REVIEW

**Review by trentmurray — APPROVED** [review]
Changes look sane.

**Comment by trentmurray** [comment]
@robinedwards  - any chance we can get a newer version out?

**Review by robinedwards — COMMENTED** [review]
I will take a look now

**Comment by MattGalvis** [comment]
I specifically put this here to figure out the tests Im still trying to figure out. any help would be awesome

**Comment by robinedwards** [comment]
It seems the credentials are wrong when connecting to the database i can see an AuthError i the trace. On import of tests/__init__.py it tries to clear the database (probably not a sensible idea of mine).

**Comment by MattGalvis** [comment]
Yea I was wondering why @robinedwards but if i were to use the change PW call there before to make it work it still gets errors i will put that up in PR now

**Comment by MattGalvis** [comment]
Really unsure how these tests were run locally before @robinedwards do u remember how?

**Comment by MattGalvis** [comment]
@robinedwards sorry I rebased the branch and didn't see the comments you made on a file so I requested for you to take another look.

**Review by trentmurray — CHANGES_REQUESTED** [review]

File: django_neomodel/apps.py
Comment (MattGalvis) [reply django_neomodel/apps.py:15]: Yes this was just to run locally

**Comment by vesnikos** [comment]
any updates on this? The updated read_settings in this PR seems to be the correct ones vs the existing version

**Comment by MattGalvis** [comment]
Nope @vesnikos still waiting on review from @robinedwards

File: .gitignore
Comment (robinedwards) [inline .gitignore:19]: Do we need test.db in here twice? DS_Store also?

File: setup.py
Comment (robinedwards) [inline setup.py:15]: Can't we set `django>=2.2` or does a later version brake it? neomodel pulls in neo4j-driver right so no need to include that here

File: setup.py
Comment (robinedwards) [inline setup.py:26]: Any reason we can't support up to 3.9?

**Review by robinedwards — CHANGES_REQUESTED** [review]

**Comment by robinedwards** [comment]
Sorry for the delay. I've just submitted the review. I think there are a
few things to tidy up before it can be merged along with fixing the travis
tests.

File: setup.py
Comment (MattGalvis) [reply setup.py:15]: thanks

File: setup.py
Comment (MattGalvis) [reply setup.py:26]: in next PR u will see in comment

File: dev
Comment (MattGalvis) [reply dev:1]: I've added this file to make contributing easy i will be updating docs to have a Contribute/testing "how to"

File: Changelog
Comment (MattGalvis) [reply Changelog:12]: yea your right

File: tests/settings.py
Comment (robinedwards) [inline tests/settings.py:27]: Sorry whats this?

File: Changelog
Comment (robinedwards) [inline Changelog:2]: Can we expand on this?

File: README.rst
Comment (robinedwards) [inline README.rst:164]: If there is no link yet can we remove this text?

File: dev
Comment (robinedwards) [inline dev:1]: I think its better just to put this in the docs as opposed to having a mysterious file people source

**Review by robinedwards — COMMENTED** [review]
A few comments need addressing, the travis build doesn't pass yet but looking good

**Comment by MattGalvis** [comment]
@robinedwards should be good to merge now

**Comment by robinedwards** [comment]
Super thank you @MattGalvis for your hard work :-)

**Pull request merged**
